MidasCivil_Adapter/AdapterActions/Execute.cs

```diff
@@ -114,10 +114,10 @@ public bool RunCommand(SaveAs command)
             Directory.CreateDirectory(newDirectory);
             string[] mcbFiles = Directory.GetFiles(m_directory, "*.mcb");
             foreach (string mcbFile in mcbFiles)
-                File.Copy(mcbFile, newDirectory);
+                File.Copy(mcbFile, Path.Combine(newDirectory, command.FileName + ".mcb"));
```

Comment:
Can you store `command.FileName`, given that it accessed three times you may as well just store the `string` and the start of the method.